packages/schema/exported-schema.json

```diff
@@ -695,6 +695,29 @@
               "value": "**yes** (with exceptions, see description)"
             }
           }
+        },
+        "lock": {
+          "description": "**INTERNAL USE ONLY**. Zapier uses this config for internal operation locking.",
+          "type": "object",
+          "required": ["key"],
+          "properties": {
+            "key": {
+              "description": "The key to use for locking. This should be unique to the operation.",
+              "$ref": "/KeySchema"
```

Comment:
The lock key uses the double curly template syntax, such as `{{bundle.inputData.spreadsheet_id}}:{{bundle.inputData.worksheet_id}}`. But `KeySchema` [doesn't allow](https://github.com/zapier/zapier-platform/blob/936f3e56fc0909f49323adfa3e008ea3ee1d95e7/packages/schema/lib/schemas/KeySchema.js#L10) curly braces. Maybe it can just be a non-empty string without restrictions?

Reply:
Agreed, we do the same in `RedirectRequestSchema.url` which also supports curly syntax for variable replacement.